app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,32 @@
+class PostsController < ApplicationController
+  before_action :logged_in_user, only: [:new, :create]
+  def new
+    @post = Post.new
+  end
+
+  def create
+    @post = Post.new(title: params[:post][:title], body: params[:post][:entry], user_id: @current_user.id)
```

Comment:
- [ ] you could be using the helper `post_params` here

- [ ] you should  [use](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/authentication/posts_create_requirement.png) the `build` would be more secure and clean, instead of introducing `@current_user.id` directly